src/views/catalogi/CatalogiDetails.vue

```diff
@@ -178,65 +178,57 @@ export default {
 			handler(newCatalogiItem, oldCatalogiItem) {
 				if (!this.upToDate || JSON.stringify(newCatalogiItem) !== JSON.stringify(oldCatalogiItem)) {
 					this.catalogi = newCatalogiItem
-					newCatalogiItem && this.fetchData(newCatalogiItem?.id)
+					if (newCatalogiItem) {
+						this.loading = true
+						catalogiStore.getCatalog(newCatalogiItem.id)
+							.then(() => {
+								this.catalogi = catalogiStore.catalogiItem
+								this.loading = false
+							})
+					}
 					this.upToDate = true
-					newCatalogiItem?.organization ? this.fetchOrganization(newCatalogiItem?.organization) : this.organization = false
+					if (newCatalogiItem?.organization) {
+						this.organizationLoading = true
+						organizationStore.getOrganization(newCatalogiItem.organization)
```

Comment:
`getOrganization` bestaat niet, gebruik `getOneOrganization`
```suggestion
						organizationStore.getOneOrganization(newCatalogiItem.organization)
```

Reply:
@SudoThijn zou jij deze kunnen oppakken?